pom.xml

```diff
@@ -113,6 +113,7 @@
         <artifactId>workflow-cps</artifactId>
         <version>2.32</version>
         <scope>test</scope>
+        <classifier>tests</classifier>
```

Comment:
I think you need to keep the original dep, _and_ add the `tests`-scoped dep. To avoid then duplicating the `version`, normally this is extracted to a property like `workflow-cps-plugin.version`.

Reply:
(still true)